flintrock/flintrock.py

```diff
@@ -268,6 +274,8 @@ def launch(
         elif spark_git_commit:
             print("Warning: Building Spark takes a long time. "
                   "e.g. 15-20 minutes on an m3.xlarge instance on EC2.")
+            if spark_git_commit == "latest":
+                spark_git_commit = get_last_commit_sha(spark_git_repository)
```

Comment:
I would find it more useful than just the sha if I needed to go trough the history on git/github to find which commit had been checked out.
But since the point of this PR is not to care about the commit being checked out, I don't really now.